deepmedic/frontEnd/configParsing/trainConfig.py

```diff
@@ -6,70 +6,76 @@
 # or read the terms at https://opensource.org/licenses/BSD-3-Clause.
 
 from __future__ import absolute_import, print_function, division
-import os
 
 from deepmedic.frontEnd.configParsing.utils import *
 from deepmedic.frontEnd.configParsing.config import Config
 
 
 class TrainConfig(Config):
     
-    #Optional but highly suggested.
+    # Optional but highly suggested.
     SESSION_NAME = "sessionName"
-    #[REQUIRED]
+    # [REQUIRED]
     FOLDER_OUTP = "folderForOutput"
     SAVED_MODEL = "cnnModelFilePath"
-    TENSORBOARD_LOG = "tensorboardLog"
+    TENSORBOARD_LOG = "tensorboardLogtensorboardLog"
```

Comment:
something wrong happened here.

```suggestion
    TENSORBOARD_LOG = "tensorboardLog"
```